static/js/components/forms/BulkEnrollmentForm.js

```diff
@@ -174,13 +201,17 @@ export class BulkEnrollmentForm extends React.Component<
                 name="product_id"
                 onChange={this.onChangeProductId(setFieldValue)}
               >
-                {R.toPairs(productMap[values.product_type]).map(
-                  ([productId, productObject]) => (
-                    <option key={productId} value={productId}>
-                      {productObject.title}
-                    </option>
-                  )
-                )}
+                {sortedProductIds.map(productId => (
+                  <option key={productId} value={productId}>
+                    {productMap[values.product_type][productId]["readable_id"]
```

Comment:
what value we can expect in `product_type`, can we add condition on `product_type `, instaed of going deep till `readable_id`